### PR TITLE
Add number formatting style guide, autogenerate ToC

### PR DESCRIPTION
**Pull request by nydr**

### CODE REVIEW

File: content/en/contribute/docs/style-guide.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ linkTitle: "Style Guide"
 weight: 10
 description: >
   Editorial guidelines for writing documentation
+show_toc: true
```
Comment (mrjones-plip) [inline content/en/contribute/docs/style-guide.md:7]: hah! love the simplicity of this and all the extra lines an manual maintenance you got rid of.  pro move!

File: content/en/contribute/docs/style-guide.md [hunk 2]
```diff
@@ -262,6 +230,19 @@ represents.
...
+
+There's a shortcode `format-number` which will replace `_` with _Thin Space_ to simplify formatting in this way.
+
+| Do                                    | Don't   |
+|---------------------------------------|---------|
+| `{{</* format-number 10_000 */>}}`    | 10,000  |
+| `{{</* format-number 10_000 */>}}`    | 10.000  |
+| `{{</* format-number 1_000_000 */>}}` | 1000000 |
+
+[^digit-grouping]: https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping
```
Comment (mrjones-plip) [inline content/en/contribute/docs/style-guide.md:244]: While I didn't know how to do citations, so far we haven't done them in our docs.  Not too big of a deal, but I think just a normal hyperlink on [line 234](https://github.com/medic/cht-docs/pull/1200/files#diff-b13c104ce4c4d34c0be07c5353173d6482a4f0b4651e437a639c03e5997abf44R234) is more inline with our existing styles.

**Review by mrjones-plip — APPROVED** [review]
thanks for pulling this together so quickly! 

one minor change that I'll pre-approve so you can zip this out live when you're ready

File: content/en/contribute/docs/style-guide.md (hunk 1 above)
Comment (nydr) [reply content/en/contribute/docs/style-guide.md:7]: One thing of note is that the layout is slightly different: 
image

Also there's already a ToC on the right-hand side, and this is the only page with a ToC

File: content/en/contribute/docs/style-guide.md (hunk 2 above)
Comment (nydr) [reply content/en/contribute/docs/style-guide.md:244]: I was already on the fence about footnotes, I'll change it to be more like the rest of the docs

File: content/en/contribute/docs/style-guide.md (hunk 1 above)
Comment (mrjones-plip) [reply content/en/contribute/docs/style-guide.md:7]: yeah - I think it's fine. Thanks for calling out the difference!

File: layouts/_default/content.html [hunk 3]
```diff
@@ -9,6 +9,9 @@ <h1>{{ .Title }}</h1>
 		{{ partial "reading-time.html" . }}
 		{{ end }}
 	</header>
+	{{ if eq (.Params.show_toc) true }}
+	{{ .TableOfContents }}
+	{{ end }}
```
Comment (jkuester) [inline layouts/_default/content.html:14]: :heart:

**Review by jkuester — APPROVED** [review]
LGTM

**Review by garethbowen — APPROVED** [review]
Thanks!

**Pull request merged**